frontend/src/metabase/querying/components/FilterValuePicker/utils.ts

```diff
@@ -30,10 +30,12 @@ export function canSearchFieldValues(
 }
 
 function getFieldOptions(fieldValues: FieldValue[]): SelectItem[] {
-  return fieldValues.map(([value, label = value]) => ({
-    value: String(value),
-    label: String(label),
-  }));
+  return fieldValues
+    .filter(([value]) => value != null)
```

Comment:
Before:
<img width="428" alt="Screenshot 2024-01-14 at 15 11 31" src="https://github.com/metabase/metabase/assets/8542534/b8ca43c0-f43b-4988-8c01-94dc373a8641">

After:
<img width="445" alt="Screenshot 2024-01-14 at 15 13 24" src="https://github.com/metabase/metabase/assets/8542534/e638c98c-ad8f-4e04-9723-cf55b36f9258">

There was an old comments about having to exclude nulls from field values, but I forgot about it.